### PR TITLE
Add list of latest uploads to home page

### LINKED ISSUE #152: Show most recently uploaded package versions on front page

### PR DESCRIPTION
**Pull request by paf31**
Fixes #152 

@hdgarrood How does this look?

screen shot 2016-08-13 at 12 30 10 pm

### CODE REVIEW

File: src/Handler/Database.hs [hunk 1]
```diff
@@ -33,6 +34,19 @@ getAllPackageNames = do
   contents <- liftIO $ getDirectoryContents (dir ++ "/verified/")
   return . sort . rights $ map mkPackageName contents
 
+getLatestPackages :: Handler [(PackageName, Version)]
+getLatestPackages = do
+    pkgNames <- getAllPackageNames
+    pkgNamesAndVersions <- traverse withTimestamp pkgNames
```
Comment (hdgarrood) [inline src/Handler/Database.hs]: I might be missing something but could this identifier's name be a little misleading?

File: templates/default-layout.lucius [hunk 2]
```diff
@@ -60,7 +60,7 @@ body {
 
 .col ul,
 .col ol {
-  padding-left: 25px;
+  padding-left: 45px;
```
Comment (hdgarrood) [inline templates/default-layout.lucius]: Do you know what else is affected by this? I think this affects display of type class members and data constructors, and perhaps also module listings on package homepages, for example. Do they still look ok? Would it make sense to use a class to make this rule a little more specific?

File: src/Handler/Database.hs (hunk 1 above)
Comment (paf31) [inline src/Handler/Database.hs]: Oops, yes. I'll fix this.

File: templates/default-layout.lucius (hunk 2 above)
Comment (paf31) [inline templates/default-layout.lucius]: I checked, and type classes etc. looked okay to me. I'm happy to revert this though.

Comment (hdgarrood) [inline templates/default-layout.lucius]: Ok, :+1: to merge then.

**Pull request closed without merging**